docs/manual-EN/2.query-language/3.language-structure/user-defined-variables.md

```diff
@@ -0,0 +1,16 @@
+# User-Defined Variables
+
+**Nebula Graph** supports the concept of user-defined variables, which allows passing of a value from one statement to another. A user-defined variable in written as `$ var_name`, `var_name` is the name of variable and can consist of alphanumeric characters, any other characters are not recommended currently.
```

Comment:
**Nebula Graph** supports user-defined variables, which allows passing the result of one statement to another.